server/plugin/command.go

```diff
@@ -340,6 +389,12 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 			}
 			subOrgMsg += "\n\n" + fmt.Sprintf("Notifications are disabled for %s", excludeMsg)
 		}
+		githubHooks, _, _ := githubClient.Repositories.ListHooks(ctx, owner, repo, &github.ListOptions{PerPage: page})
+		if len(githubHooks) == 0 {
+			subOrgMsg += "\nNo webhook was found for this repository or organization. Would you like the webhook to be created?"
+			subOrgMsg += "\nSuggested Command to create a wehook"
+			subOrgMsg += "\n`/github webhook add " + owner + "[content-type:application-json/application/x-www-form-urlencoded] [features:create/delete..]  [ssecure-ssl:true/false]`"
```

Comment:
We should only be showing the standard command. Also, please put this text in a shared function to be used in the other case where we output this information.